js/main.js

```diff
@@ -21,51 +39,127 @@ class Awesomebooks {
-function sendToLocal(a, b) {
-  localStorage.setItem(a, JSON.stringify(b));
-  Awesomebooks.createBooks();
-}
+
+// listens to form submission event
 document.getElementById('add-book-form').addEventListener('submit', (e) => {
   e.preventDefault();
-  const nextBookId = bookData.length + 1;
 
+  const bookId = Awesomebooks.getAddedBookId();
   const bookObject = new Awesomebooks(
-    nextBookId,
+    bookId,
     document.getElementById('title').value,
     document.getElementById('author').value,
   );
   Awesomebooks.addBook(bookObject);
   document.getElementById('title').value = '';
   document.getElementById('author').value = '';
 });
-Awesomebooks.delBook();
+
+// Add active class to the current button (highlight it)
+// var header = document.querySelector('.nav-right__list-container');
+const links = document.getElementsByClassName('nav__item');
+for (let i = 0; i < links.length; i += 1) {
+  links[i].addEventListener('click', function () {
+    const current = document.getElementsByClassName('active');
+    current[0].className = current[0]?.className?.replace(' active', '');
+    this.className += ' active';
+  });
+}
+
+/* eslint-disable-next-line no-unused-vars */
+function toggleSection(domId) {
+  if (domId === 'books-list') {
+    document.getElementById('add-books').style.display = 'none';
+    document.getElementById('contact').style.display = 'none';
+    document.getElementById('books-list').style.display = 'block';
+  }
+
+  if (domId === 'add-books') {
+    document.getElementById('books-list').style.display = 'none';
+    document.getElementById('contact').style.display = 'none';
+    document.getElementById('add-books').style.display = 'flex';
+  }
+
+  if (domId === 'contact') {
+    document.getElementById('books-list').style.display = 'none';
+    document.getElementById('add-books').style.display = 'none';
+    document.getElementById('contact').style.display = 'block';
+  }
+}
+
+// function timeConversion12Hours(dt) {
+//   //slice time for conversion
+//   let ot = dt.slice(11, 16);
+//   let mt = dt.slice(11, 16);
+
+//   /* let lastTwo = mt.slice(-2); */
+//   let firstTwo = mt.slice(0, 2);
+//   let intFirstTwo = parseInt(firstTwo);
+//   let timeZeroIndex = mt.slice(0, 1);
+
+//   /* console.log("ZERO INDEX VALUE IS => ", timeZeroIndex); */
+
+//   if (intFirstTwo == '00') {
+//     intFirstTwo += 12;
+//     mt = mt.replace(firstTwo, intFirstTwo) + 'AM';
+//   } else if (intFirstTwo < 12 && intFirstTwo != '00' && timeZeroIndex != 0) {
+//     mt = mt + 'AM';
+//   } else if (intFirstTwo < 12 && intFirstTwo != '00' && timeZeroIndex == 0) {
+//     mt = mt.replace('0', '') + 'AM';
+//   } else if (intFirstTwo > 12) {
+//     intFirstTwo -= 12;
+//     mt = mt.replace(firstTwo, intFirstTwo) + 'PM';
+//   } else {
+//     if (intFirstTwo == 12) {
+//       mt = mt + 'PM';
+//     }
+//   }
+
+//   dt = dt.replace(ot, mt);
+//   return dt;
+// }
+
+// function sliceDatetime(dt) {
+//   return dt.slice(0, 16);
+// }
```

Comment:
- [ ] it would be better if you remove the unused commented code in order to keep your file clean and pro

Reply:
Tank you sir